cdragontoolbox/tftdata.py

```diff
@@ -108,29 +165,44 @@ def parse_items(self, map22):
 
         return items
 
+    def find_closest_path(self, parent, name):
+        test = name
+        while test:
+            path = os.path.join(parent, test, test + ".bin")
+            if os.path.exists(path):
+                return path
+
+            test = test[:-1]
+
+        return ""
```

Comment:
You should return `None` for "nothing", or even raise an error since the case does not seem to be handled anyway.